libs/components/src/lib/button/button.spec.ts

```diff
@@ -184,6 +184,82 @@ describe('vwc-button', () => {
 
 	});
 
+	describe('link', () => {
```

Comment:
IMO there should be one test for href here, which checks that when href is set, the control turns into an anchor tag

```suggestion
	describe('href', () => {
```